src/main/java/bio/terra/cli/command/resource/create/GcpNotebook.java

```diff
@@ -65,9 +65,11 @@
   private String postStartupScript;
 
   @CommandLine.Option(
-      names = "--metadata",
+      names = "-M, --metadata",
       description =
-          "Custom metadata to apply to this instance.\nBy default sets some jupyterlab extensions "
+          "Custom metadata to apply to this instance.\n"
+              + "specify multiple metadata in the format of --metadata=key1=value1 --Mkey2=value2.\n"
```

Comment:
should that be `-M` instead of `--M`?

Reply:
i'm following this example https://picocli.info/#_key_only_map_parameters and they use one dash for the short version and two dash for the full name.